microsoft/testsuites/acc/cvm.py

```diff
@@ -26,6 +26,21 @@ class CVMSuite(TestSuite):
         r"(?P<config_a>(0x[a-z,A-Z,0-9]+)), Group B.(?P<config_b>(0x[a-z,A-Z,0-9]+))"
     )
 
+    # HYPERV_CPUID_ISOLATION_CONFIG.EBX Structure
+    #     UINT32 ParavisorPresent : 1;
+    #     UINT32 Reserved0 : 31;
+    HV_PARAVISOR_PRESENT_MASK = 1
+
+    # HYPERV_CPUID_ISOLATION_CONFIG.EBX Structure
```

Comment:
the structure name is the same with line 29?

Reply:
Whoops, the structure that holds the paravisor bit is actually HYPERV_CPUID_ISOLATION_CONFIG.EAX

Thanks for catching this.